attest/ake/src/lib.rs

```diff
@@ -113,7 +116,7 @@ mod test {
 
         // initiator = authpending, responder = ready
 
-        let auth_response_input = AuthResponseInput::new(auth_response_output, identities);
+        let auth_response_input = AuthResponseInput::new(auth_response_output, identities, None);
```

Comment:
A None value for time is used in tests because skipping time verification is already tested in mc-attestation-verifier.